scripts/custom-screens/CustomFormControls.tsx

```diff
@@ -14,11 +14,14 @@ import {
   TimeField,
   TextArea,
   MaskedField,
-  SliderField
+  SliderField,
+  SourceCodeEditor
 } from "@haulmont/jmix-react-antd";
 import { FormattedMessage, useIntl } from "react-intl";
 import { useForm } from "antd/es/form/Form";
 import dayjs from "dayjs";
+import "ace-builds/src-noconflict/mode-javascript";
+import "ace-builds/src-noconflict/theme-github";
```

Comment:
Can we avoid having to make these imports manually?

Reply:
How is this resolved? I still see the import.

Reply:
What are these imports for?

Reply:
OK, I see, it seems to be the way `react-ace` works.

Please add information to QA Notes about what imports needs to be added in order for the component to work.

Reply:
Yes, without these scripts the code editor will not work. They are needed for syntax highlighting and editor theme.
User sees only the script for highlighting the language now.